mongodbatlas/resource_mongodbatlas_cluster_migration_test.go

```diff
@@ -1,130 +0,0 @@
-package mongodbatlas
```

Comment:
This file has not deleted but has been renamed to `mongodbatlas/resource_mongodbatlas_cluster_test.go` since it contains unit tests